Apps/Examples/Examples/All Examples/SwitchStylesExample.swift

```diff
@@ -22,48 +50,33 @@ public class SwitchStylesExample: UIViewController, ExampleProtocol {
 
         // Add a `UISegmentedControl` to toggle between Mapbox Streets, Mapbox Satellite Streets, and a custom style.
         addStyleToggle()
+        installConstraints()
     }
 
     @objc func switchStyle(sender: UISegmentedControl) {
-        switch sender.selectedSegmentIndex {
-        case 0:
-            // Set the map's style to the default Mapbox Streets style.
-            // The map's `StyleURI` can be set to default Mapbox styles: `.light`, `.dark`, `.satellite`, `.satelliteStreets`, and `.outdoors`.
-            mapView.mapboxMap.style.uri = .streets
-        case 1:
-            // Set the map's style to Mapbox Satellite Streets.
-            mapView.mapboxMap.style.uri = .satelliteStreets
-        case 2:
-            // Blueprint, a custom style designed by Amy Lee Walton. https://www.mapbox.com/gallery/#blueprint
-            // Load the local style JSON. It conforms to the Mapbox Style Specification <https://docs.mapbox.com/mapbox-gl-js/style-spec/>
-            let localStyleURL = Bundle.main.url(forResource: "blueprint_style", withExtension: "json")!
-
-            // Set the map's style URI to a custom style URL.
-            mapView.mapboxMap.style.uri = StyleURI(url: localStyleURL)
-
-            // A Mapbox Studio style can also be loaded as a `StyleURI`.
-            // let studioStyleURL = URL(string: "mapbox://styles/examples/ckox18pjy140u17pdvmpgut4i")!
-        default:
-            mapView.mapboxMap.style.uri = .satelliteStreets
-        }
+        guard let style = Style(rawValue: sender.selectedSegmentIndex) else { return }
+
+        mapView.mapboxMap.style.uri = style.uri
```

Comment:
There is should be a variable of type `Style` which one you have to update when a user selects another style in the segmented control. 
```suggestion
        style = Style(rawValue: sender.selectedSegmentIndex) ?? . satelliteStreets
```